src/main/webapp/web.xml

```diff
@@ -7,8 +7,88 @@
         <servlet-name>index</servlet-name>
         <servlet-class>mate.controller.IndexController</servlet-class>
     </servlet>
+    <servlet>
```

Comment:
I prefer to use annotations such as WebServlet and WebFilter, but it's up to you)